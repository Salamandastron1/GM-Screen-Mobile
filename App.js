import React from 'react';
import { createStackNavigator, createAppContainer } from 'react-navigation';
import LandingPage from './components/LandingPage';
import CodeForm from './components/CodeForm';
import CharacterForm from './components/CharacterForm';
import PlayerScreen from './components/PlayerScreen';

const RootStack = createStackNavigator(
  {
    LandingPage,
    CodeForm,
    CharacterForm,
    PlayerScreen,
  },
  {
    initialRouteName: 'LandingPage',
    headerMode: 'none',
  },
);

const AppContainer = createAppContainer(RootStack);

const App = () => (
  <AppContainer />
);

export default App;
