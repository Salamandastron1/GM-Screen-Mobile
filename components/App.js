import React from 'react';
import { createStackNavigator, createAppContainer } from 'react-navigation';
import LandingPage from './LandingPage';
import CodeForm from './CodeForm';
import CharacterForm from './CharacterForm';
import PlayerScreen from './PlayerScreen';

const RootStack = createStackNavigator(
  {
    LandingPage,
    CodeForm,
    CharacterForm,
    PlayerScreen,
  },
  {
    initialRouteName: 'LandingPage',
    headerMode: 'none',
  },
);

const AppContainer = createAppContainer(RootStack);


export default class App extends React.Component {
  constructor() {
    super();
    this.state = {

    };
  }

  render() {
    return (
      <AppContainer />
    );
  }
}
