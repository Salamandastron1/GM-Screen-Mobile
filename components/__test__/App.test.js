/* eslint-disable */

import React from 'react';
import { shallow, mount } from 'enzyme';
import App from '../App';

it('should run', () => {
  const tree = shallow(<App />);
  expect(tree).toMatchSnapshot();
});