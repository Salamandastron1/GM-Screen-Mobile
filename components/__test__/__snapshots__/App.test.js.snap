// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the shapshot 1`] = `<NavigationContainer />`;
